src/Test/Factories.php

```diff
@@ -57,6 +60,7 @@ public static function _setUpFactories(): void
      * @internal
      * @after
      */
+    #[After]
```

Comment:
having both the attribute && the doc-comment `@after` does not trigger a deprecation in PHPUnit 11.